src/sentry/static/sentry/app/components/errors/groupEventDetailsLoadingError.tsx

```diff
@@ -1,18 +1,22 @@
-import PropTypes from 'prop-types';
 import React from 'react';
 
-import SentryTypes from 'app/sentryTypes';
 import {t} from 'app/locale';
+import {Environment} from 'app/types';
 import DetailedError from 'app/components/errors/detailedError';
 
-const GroupEventDetailsLoadingError = ({onRetry, environments}) => {
+type Props = {
+  environments: Environment[];
+  onRetry?: (e: React.MouseEvent) => void;
```

Comment:
is it not possible to add what type of mouse event will be used? example `HTMLDivElement`

Reply:
The underlying component used plain React.MouseEvent, but I changed it in both files 👍